frontend/redux/nodes/entities/campaigns/helpers.js

```diff
@@ -26,6 +49,14 @@ const updateCampaignStateFromResults = (campaign, { data }) => {
       failed: newFailed,
       total: newTotal,
     };
+    newErrors = [
+      ...errors,
+      {
+        host_hostname: host.hostname,
+        osquery_version: host.osquery_version,
+        error: 'unknown error: upgrade osquery on this host to a version > 4.4.0 to see a more descriptive message.',
```

Comment:
Maybe can compact this some? such as
```suggestion
        error: 'upgrade osquery on this host to 4.4.0+ for error details',
```